tests/test_API.py

```diff
@@ -1,13 +1,12 @@
 
 from base import *
-
-
+import vcr
```

Comment:
cool!  The name of this library + it's use case with Twilio (audio) really threw me off.  This is a neat one!